meteor-app/imports/startup/server/email.js

```diff
@@ -0,0 +1,42 @@
+import { Meteor } from 'meteor/meteor'
+import { Email } from 'meteor/email'
+const debug = require('debug')('att:email')
+import { eventLog } from '/imports/api/events'
+import log from '/imports/lib/log'
+
+const DEFAULT_MESSAGE = 'Hello from back2bikes. Heres your pin'
+const DEFAULT_DESTINATION = 'mrslwiseman@gmail.com'
+const DEFAULT_SUBJECT = 'Back 2 Bikes PIN reminder.'
+
+const emailSettings = { enabled: false }
+
+Meteor.startup(() => {
+  if (Meteor.settings.env && Meteor.settings.env.MAIL_URL) {
+    emailSettings.enabled = Meteor.settings.env.MAIL_URL != undefined
+    debug(`Email sending is ${emailSettings.enabled}`)
+  }
+})
+
+Meteor.methods({
```

Comment:
The guts of this don't do much, I would be tempted to dispense with it completely and call the function directly from where it is being called.